javascript-sdk/application-examples/submit-form/submit-form.js

```diff
@@ -0,0 +1,43 @@
+const { wrap, configure } = require('agentql');
+const { chromium } = require('playwright');
+
+(async () => {
+  configure({ apiKey: 'YOUR_API_KEY' });
```

Comment:
We're planning to remove this from example file.

Reply:
Wondering why this would be removed, seems useful for copy and paste purposes to keep?